make_dataset.py

```diff
@@ -483,25 +500,60 @@ def parse_args(args_input=None):
             warnings.warn("Unable to degrade chosen excerpt from "
                           f"{comp.csv_path} and no clean excerpts requested."
                           " Skipping.", UserWarning)
-
+    
     meta_file.close()
+    
+    fh_dict = {
+        split: open(
+            os.path.join(ARGS.output_dir, f'{split}_cmd_corpus.csv'
+        ), 'w') for split in ['train', 'valid', 'test']
+    }
+    line_counts = {
+        split: 0 for split in ['train', 'valid', 'test']
+    }
```

Comment:
This is already in current_splits, but not explicitly as a dict, and this doesn't add much time anyways I'm sure.